### PR TITLE
Long Term Support (LTS) version of Blender

### PR DESCRIPTION
**Pull request by walles**
This Blender LTS version will receive bugfixes until Q1 2022:
<https://www.blender.org/download/lts/>

The `.rb` file is copied from here:
<https://github.com/Homebrew/homebrew-cask/blob/96ce52a9ef8af1c3d5e4c47a051f1a903fa4eaf5/Casks/blender.rb>

Requested in this ticket:
<https://github.com/Homebrew/homebrew-cask/issues/88481>

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask-versions/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask-versions/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by walles** [comment]
```
~/s/homebrew-cask-versions (walles/blender-lts|✔) [1] $ brew cask audit --download Casks/blender-lts.rb
==> Downloading https://download.blender.org/release/Blender2.83/blender-2.83.5-macOS.dmg
######################################################################## 100.0%
==> Verifying SHA-256 checksum for Cask 'blender-lts'.
audit for blender-lts: passed
~/s/homebrew-cask-versions (walles/blender-lts|✔) $
```

File: Casks/blender-lts.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+cask "blender-lts" do
+  version "2.83.5"
+  sha256 "00a8e6ef52b84256ab64d33df3a079ebe5ac743aead06b74cd99987382be7f52"
+
+  url "https://download.blender.org/release/Blender#{version.major_minor.delete("a-z")}/blender-#{version}-macOS.dmg"
+  appcast "https://download.blender.org/release/",
+          must_contain: version.major_minor.delete("a-z")
+  name "Blender"
+  desc "Open source 3D creation. Free to use for any purpose, forever."
```
Comment (walles) [inline Casks/blender-lts.rb:9]: `desc` taken from the image caption at the top of <https://blender.org>.

Comment (reitermarkus) [reply Casks/blender-lts.rb:9]: I think the description from the “Get Involved” card is a bit more descriptive:

“free and open source 3D creation suite”

Comment (reitermarkus) [reply Casks/blender-lts.rb]: Actually, the stable `blender` cask already has a description, so let's use the exact same one:

`Free and open-source 3D creation suite`

Comment (walles) [reply Casks/blender-lts.rb]: Done

**Comment by walles** [comment]
As far as I know this should be ready to merge.

Thanks for your fixup and opinion so far @reitermarkus!

**Comment by reitermarkus** [comment]
Thanks, @walles! Please also submit a PR with the corresponding `conflicts_with` for the stable cask.

**Pull request merged**